lib/concerns/traceable_tag.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module TraceableTag
```

Comment:
Am I understanding correctly that each Job class would then import this

Reply:
Yes, each job includes it and sets the service in the same manner as the controllers. The shared logic moves here, and the Traceable concern for controllers now includes this and just adds the code to wire up the before action.